modules/bridge/src/main/ts/ephox/bridge/components/dialog/Label.ts

```diff
@@ -7,16 +7,19 @@ export interface LabelSpec {
   type: 'label';
   label: string;
   items: BodyComponentSpec[];
+  align?: 'start' | 'end';
```

Comment:
I'm happy to add `center`, if you think it's valuable

Reply:
Center is the default when the label is in a grid component, as in the ai dialog. I forgot about that, but we should probably implement a forced alignment for start as well, and center could be added for completeness.

Reply:
The label is still has start text alignment in a grid component, so there is no concern making `align: 'start'` the default. I have also added center alignment.